### PR TITLE
feat: release 0.31.0

### PR DESCRIPTION
**Pull request by magnus-madsen**

### CODE REVIEW

**Comment by magnus-madsen** [comment]
@mlutze Anything important missed?

**Comment by mlutze** [comment]
Should we note the two backward-compatible syntax changes (records and imports)?

**Comment by mlutze** [comment]
Looks like there are some ergonomic things with unit tests ?

**Comment by magnus-madsen** [comment]
> Looks like there are some ergonomic things with unit tests ?

Which?

**Comment by mlutze** [comment]
> > Looks like there are some ergonomic things with unit tests ?
> 
> Which?

The addition of `Assert.eq` for a more imperative testing style.

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+Version 0.31.0:
+- Language: Added new object expressions (thanks Paul!)
+- Language: Added `par` expression (thanks Jakob!)
+- Language: Added safe `upcast` expression (thanks Jakob!)
+- Language: Added support for top-level imports (thanks Magnus!)
+- Language: Renamed `Impure` to `IO` (thanks Matt!)
+- Language: Changed effect syntax (thanks Matt!)
+  - old syntax: `& Pure`, new syntax: `\ IO`.
```
Comment (mlutze) [inline docs/CHANGELOG.md:10]: nitpick: maybe `& Impure` to match the `\ IO`

**Pull request merged**